src/implementations/vanilla/components/inpage-navigation/_inpage-navigation.scss

```diff
@@ -29,6 +29,7 @@ $_outline: null !default;
     display: block;
     position: sticky;
     top: 0;
+    z-index: 0;
```

Comment:
mmh..i'm not sure we want to set z 0 z-index on an element like the inpage navigation. This would only be safe in case nothing else, basically, is present in the left sidebar, otherwise any element could easily go on top of the inpage, it's enough a relative positioning for this to happen.

Reply:
that's true.. I'll test another solution